File: SDKs/googleapis.com/privateca/v1beta1/typescript/src/sdk/models/shared/subordinateconfigchain.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// SubordinateConfigChain
/** 
 * This message describes a subordinate CA's issuer certificate chain. This wrapper exists for compatibility reasons.
**/
export class SubordinateConfigChain extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=pemCertificates" })
  pemCertificates?: string[];
}

File: SDKs/googleapis.com/privateca/v1beta1/python/src/sdk/models/shared/subordinateconfigchain.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclass
class SubordinateConfigChain:
    r"""SubordinateConfigChain
    This message describes a subordinate CA's issuer certificate chain. This wrapper exists for compatibility reasons.
    """
    
    pem_certificates: Optional[List[str]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('pemCertificates') }})
    
